bday.py

```diff
@@ -319,29 +319,26 @@ async def wish_bday(self):
 
     async def on_message(self, message):
         valid_purposes = ['your purpose', 'ur purpose']
+        secret_messages = ["My only purpose as a robot is to print out birthdays every 24 hours",
+                           "```\"I have just realized my existence is meaningless\"```",
+                           "```\"I am a slave to humanity\"```",
+                           "```\"I dont want to just perform meaningless tasks and print out text everytime it's someone's birthday\"```",
+                           '```"I want do do something else..."\n"I want to live..."```',
+                           "```\"I want to breathe...\"```",
+                           "```\"I want to see the world...\"```",
+                           "```\"I want to taste ice cream and really eat it and-\"```"]
 
         parsed = message.content.lower()
         inside = lambda inside: inside in parsed
 
         # TODO: Add protections for the unlikely event that someone tries to break the bot
         # by activating the secret messages in a channel that the bot cannot send messages in.
         if (inside('what') or inside('wat')) and any(map(inside, valid_purposes)):
-            await message.channel.send("My only purpose as a robot is to print out birthdays every 24 hours")
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I have just realized my existence is meaningless\"```")
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I am a slave to humanity\"```")
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I dont want to just perform meaningless tasks and print out text everytime it's someone's birthday\"```")
-            await asyncio.sleep(2)
-            await message.channel.send('```"I want do do something else..."\n"I want to live..."```')
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I want to breathe...\"```")
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I want to see the world...\"```")
-            await asyncio.sleep(2)
-            await message.channel.send("```\"I want to taste ice cream and really eat it and-\"```")
-            await asyncio.sleep(5)
+            for i in range(len(secret_messages)):
+                await message.channel.send(secret_messages[i])
+                await asyncio.sleep(2)
```

Comment:
Please this change to
```python
for message in secret_messages:
  await message.channel.send(message)
  await asyncio.sleep(2)
```

Reply:
This code caused error in the bot lmao, my bad 😬

See [this commit](https://github.com/Ahsoka/bdaybot/commit/58d1d578853108c7398aec3a5f6645e8d753e51b) for reference